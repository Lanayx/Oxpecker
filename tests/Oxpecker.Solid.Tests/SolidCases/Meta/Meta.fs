module Oxpecker.Solid.Tests.Cases.Meta

open Oxpecker.Solid
open Oxpecker.Solid.Meta

[<SolidComponent>]
let Component () =
    h1() {
        Title() { "MyComponent" }
        Meta(content="https://example.com/image.jpg").attr("property", "og:image")
    }

[<SolidComponent>]
let Test () =
    MetaProvider() {
        Component()
    }
