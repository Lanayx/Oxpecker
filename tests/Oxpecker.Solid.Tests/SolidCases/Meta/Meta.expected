import { MetaProvider, Meta, Title } from "@solidjs/meta";

export function Component() {
    return <h1>
        <Title>
            MyComponent
        </Title>
        <Meta content="https://example.com/image.jpg"
            property="og:image"></Meta>
    </h1>;
}

export function Test() {
    return <MetaProvider>
        {Component()}
    </MetaProvider>;
}

